### PR TITLE
pakcs: 1.14.0 -> 2.0.2

### PR DESCRIPTION
**Pull request by ghost**
###### Motivation for this change

Thanks @meditans and @cleverca22 for hints
Closes https://github.com/NixOS/nixpkgs/issues/44176

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg eval

**Comment by FRidh** [comment]
@GrahamcOfBorg eval

**Comment by cleverca22** [comment]
@GrahamcOfBorg eval

**Comment by cleverca22** [comment]
@GrahamcOfBorg build

**Comment by samueldr** [comment]
@GrahamcOfBorg build pakcs

**Comment by gnidorah** [comment]
> Failure on x86_64-linux

Looks like too long task for ofborg `building of '/nix/store/0qgyy754cvsdg57s0kfvapjd4pd4nl2w-ghc-8.4.3.drv' timed out after 1800 seconds`

File: pkgs/development/compilers/pakcs/default.nix [hunk 1]
```diff
@@ -1,139 +1,72 @@
...
-    rm -r frontend
-
-    # Prevent embedding the derivation build directory as temp.
-    export TEMP=/tmp
-
-    # Copy to in place build location and run the build.
+    mkdir -p $out/pakcs
     cp -r * $out/pakcs
-    (cd $out/pakcs ; make)
+    (cd $out/pakcs ; make -j$NIX_BUILD_CORES $makeFlags)
```
Comment (Infinisil) [inline pkgs/development/compilers/pakcs/default.nix]: What's this needed for exactly? Does a normal `make` not work?

Comment (gnidorah) [reply pkgs/development/compilers/pakcs/default.nix]: @Infinisil  It works, but swiProlog still embedds references to build directory, so some utilities doesn't work. See upper comment:
```
cypm new: EXISTENCE ERROR: source_sink "/tmp/nix-build-pakcs-2.0.2.drv-0/pakcs-2.0.2/currytools/cpm/templates/LICENSE" does not exist
```

**Comment by xeji** [comment]
let's try again
@GrahamcOfBorg  build pakcs

**Comment by gnidorah** [comment]
@xeji Thank you!

**Pull request merged**